src/interfaces/PushData.ts

```diff
@@ -21,15 +26,21 @@ export interface IDrillableItemPushData {
     title: string;
     attributes: IAttributeDisplayFormUri[];
 }
-
 export interface IPushData {
     result?: Execution.IExecutionResponses;
     properties?: {
         sortItems?: AFM.SortItem[];
         totals?: VisualizationObject.IVisualizationTotal[];
         controls?: IVisualizationPropertiesControls;
     };
+    sanitizedProperties?: boolean;
+    initialProperties?: {
```

Comment:
whats the difference between `properties` and `initialProperties`? it seems to contain completely different data, but it's both called "properties"

Reply:
Its just adding types into structure that is used more than year 

https://github.com/gooddata/gooddata-react-components/blob/baef9be455dd2f3224570ed52392899d293c053a/src/internal/components/pluggableVisualizations/baseChart/PluggableBaseChart.tsx#L325